src/main/java/eu/europeana/apikey/config/SecurityConfig.java

```diff
@@ -16,13 +16,15 @@
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http.requiresChannel().requestMatchers(r -> r.getHeader("X-Forwarded-Proto") != null)
-            .requiresSecure().and()
-            .authorizeRequests().antMatchers(HttpMethod.OPTIONS, "/apikey/captcha").permitAll().and()
-            .authorizeRequests().antMatchers(HttpMethod.POST, "/apikey/captcha").permitAll().and()
-            .authorizeRequests().antMatchers(HttpMethod.POST, "/apikey/validate").permitAll().and()
-            .authorizeRequests().antMatchers(HttpMethod.POST, "/apikey", "/apikey/").authenticated().and()
-            .authorizeRequests().antMatchers("/apikey/**").authenticated().and()
+        http
+            // https://stackoverflow.com/questions/46870411/how-to-require-ssl-in-some-environments-using-spring-boot-2-0-0-m4
+//                .requestMatchers( r -> r.getHeader("X-Forwarded-Proto") != null).requiresSecure().and()
+            .authorizeRequests()
+                .antMatchers(HttpMethod.DELETE,"/user/account").permitAll()
```

Comment:
Still wondering if we shouldn't change the DELETE /user/account to "authenticated" instead of "permitAll"